### PR TITLE
OSOE-56: Open-source Content Editors

### PR DESCRIPTION
**Pull request by barthamark**
[OSOE-56](https://lombiq.atlassian.net/browse/OSOE-56)

[OSOE-56]: https://lombiq.atlassian.net/browse/OSOE-56?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Lombiq.ContentEditors/Lombiq.ContentEditors.csproj [hunk 1]
```diff
@@ -19,8 +21,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Include="Assets\Scripts\async-editor.js">
+      <CopyToPublishDirectory>Never</CopyToPublishDirectory>
+    </None>
     <None Include="License.md" Pack="true" PackagePath="" />
     <None Include="NuGetIcon.png" Pack="true" PackagePath="" />
+    <None Remove="Assets\Scripts\async-editor\async-editor.js" />
```
Comment (0liver) [inline Lombiq.ContentEditors/Lombiq.ContentEditors.csproj]: These changes look undesired. Did you move the _async-editor.js_ file out of the _async-editor_ folder into _Assets\Scripts_?

Then you should be able to revert these changes here.

Comment (barthamark) [reply Lombiq.ContentEditors/Lombiq.ContentEditors.csproj]: You are correct, Rider sometimes puts stuff to the csproj after I add or remove files. Thank you for pointing out!

**Comment by jtkech** [comment]
@barthamark 

Just for info we already needed to upgrade Folkways to OC 1.5.0, see https://github.com/Smithsonian/CFCH-folkwaysCore/pull/92#issuecomment-1460362992, and here we did something related #35 based on the `issue/OSOE-56` branch. So if you can take a look and consider what has been done in #35.

We also did something in the OSOCE repo, see https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/issues/424 and https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/426, but based on the dev branch, I didn't see that an `issue/OSOE-56` branch also exists in this other repo ;) Anyway, as I remember I didn't do anything special in the OSOCE repo, so just to take a look.

**Comment by barthamark** [comment]
@jtkech I was a bit overenthusiastic yesterday and started doing the same (upgrade to 1.5.0 and upgrade to Node.JS Extensions) then I noticed you are doing something similar, so I abandoned it. 🙂 I'll leave it to you to finish what you started, I'll address my issue after that.

**Comment by jtkech** [comment]
@barthamark

No problem, I already did in #35 what we needed, so you can finalize here what should be done at the end, just take a look at what I did and could be used here.

From memory I removed from the repo `package-lock.json` but also `pnpm-lock.yaml` and all the `wwwroot` folder that all are generated on build, and added them to `.gitignore` so that we have less diffs (package.json files remain) for a project using this submodule.
